enarx-keep-sgx-shim/src/start.S

```diff
@@ -162,7 +197,30 @@ syscall:
     mov     0x48(%rsp),             %rsp            # Get the exit context
```

Comment:
You `MUST NOT` modify the stack after this point. I'm honestly not sure how the below code even works. We need to save the extended CPU state immediately before or after the `savep` macro above.

Reply:
I've relocated the code saving extended CPU state to immediately before `savep`.